sql/opt/exec/explain: TestContentionTimeOnWrites failed

sql/opt/exec/explain.TestContentionTimeOnWrites [failed](https://mesolite.cluster.engflow.com/invocations/default/e74dce74-882c-4e77-a802-ec5c6f0e5f23?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL29wdC9leGVjL2V4cGxhaW46ZXhwbGFpbl90ZXN0) on release-25.3.2-rc @ [a5e060ee11ad11e1649e7569486b579e9da8c8d7](https://github.com/cockroachdb/cockroach/commits/a5e060ee11ad11e1649e7569486b579e9da8c8d7):


```
=== RUN   TestContentionTimeOnWrites
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestContentionTimeOnWrites2866270563
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    output_test.go:342: 
        	Error Trace:	pkg/sql/opt/exec/explain/output_test.go:342
        	Error:      	Received unexpected error:
        	            	pq: restart transaction: TransactionRetryWithProtoRefreshError: WriteTooOldError: write for key /Tenant/10/Table/104/1/1/0 at timestamp 1757578160.942268678,0 too old; must write at or above 1757578161.043264557,1: "sql txn" meta={id=518d5236 key=/Tenant/10/Table/104/1/1/0 iso=Serializable pri=0.00182898 epo=0 ts=1757578161.043264557,1 min=1757578160.942268678,0 seq=1} lock=true stat=PENDING rts=1757578160.942268678,0 gul=1757578161.442268678,0 obs={n1@1757578161.020221371,0}
        	Test:       	TestContentionTimeOnWrites
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestContentionTimeOnWrites2866270563
--- FAIL: TestContentionTimeOnWrites (7.13s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestContentionTimeOnWrites.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-54396

--- a/pkg/sql/opt/exec/explain/output_test.go
+++ b/pkg/sql/opt/exec/explain/output_test.go
@@ -231,6 +231,8 @@ func TestContentionTimeOnWrites(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderDuress(t, "see issue #153394")
+
 	ctx := context.Background()
 	s, conn, _ := serverutils.StartServer(t, base.TestServerArgs{})
 	defer s.Stopper().Stop(ctx)

